mass_mailing_list_dynamic/views/mail_mass_mailing_list_view.xml

```diff
@@ -27,6 +28,14 @@
                                 type="object"
                                 string="Sync now"
                                 icon="fa-refresh"
```

Comment:
I did a functional test and this one is a really good fix! Improves UX very much :blush: 

However I think here the icon should be `fa-exclamation-triangle`, and in the other button below, `fa-refresh`. What do you think?

Reply:
Sure, that's better. There it goes with tests for coverage

Reply:
Also I refined behaviour to track `is_synced` only if the list is dynamic